.github/workflows/test.yml

```
@@ -15,7 +15,7 @@ jobs:
         timeout-minutes: 20
         strategy:
             matrix:
-                python-version: ["3.7", "3.8", "3.9", "3.10"]
+                python-version: ["3.8", "3.9", "3.10", "3.11"]
```

```suggestion
                python-version: ["3.8", "3.9", "3.10"]
```

pytorch lightning does not support 3.11, let's make a separate PR for that after this one just to track this